components/sensor/mopeka_std_check.rst

```diff
@@ -0,0 +1,150 @@
+Mopeka Standard Check BLE Sensor
+================================
+
+.. seo::
+    :description: Instructions for setting up Mopeka Standard Check bluetooth-based sensors in ESPHome.
+    :image: mopeka_std_check.jpg
+    :keywords: Mopeka, Mopeka Standard Check, Mopeka Std Check, BLE, Bluetooth
+
+The ``mopeka_std_check`` sensor platform lets you track the output of Mopeka
+Standard Check LP Bluetooth Low Energy devices using the 
+:doc:`/components/esp32_ble_tracker`. This component will track the tank level, 
+distance, temperature, and battery percentage of a Mopeka Standard Check LP BLE 
+device every time the sensor sends out a BLE broadcast.
+
+.. warning::
+
+    This sensor component only supports the Mopeka Standard Check devices.
+    The Mopeka Pro Check sensors are not supported.
```

Comment:
Should we add a line here to indicate that the pro check sensor is supported by the other component?

Reply:
And probably update the pro check to indicate at same spot that std sensor is supported with this component.

Reply:
Good idea. Done.